src/Components/ImagesTable/Release.tsx

```diff
@@ -26,6 +26,7 @@ const Release = ({ release }: ReleaseProps) => {
     'rhel-93': 'RHEL 9.3',
     'rhel-94': 'RHEL 9.4',
     'rhel-10-nightly': 'RHEL 10',
+    'rhel-10-beta': 'RHEL 10 Beta',
```

Comment:
Amazing, I just ran into this problem after updating the image builder API. Thanks for addressing this!:)